client/web/src/enterprise/batches/settings/CodeHostConnectionNode.tsx

```diff
@@ -64,18 +65,28 @@ export const CodeHostConnectionNode: React.FunctionComponent<CodeHostConnectionN
                         )}
                         <Icon className="icon-inline mx-2" /> {node.externalServiceURL}
                     </h3>
+                    {!isEnabled && node.credential?.isSiteCredential && (
+                        <span className="text-primary mx-3">
+                            <InfoCircleIcon className="icon-inline" /> Changesets on this code host will be created with
+                            a global token until a personal access token is added.
```

Comment:
```suggestion
                            the configured global credentials until a personal access token is added.
```

(Could we make "a personal access token is added" a link to the user settings page for credentials?)

Reply:
This is only visible on the user credentials page and let's the user know that it'll work, but they can still define their own credential to use it in their name.